conf/nginx.conf.server

```diff
@@ -150,8 +149,8 @@ server {
         # These timeouts ensure that the stale cache is used quickly if network
         # goes down. The timeouts are for time to first byte, not entire request duration.
         proxy_connect_timeout 1s;
-        proxy_read_timeout 2s;
-        proxy_send_timeout 2s;
+        proxy_read_timeout 4s;
```

Comment:
Bumped this up because was already running into issues with some endpoints that have long TTFBs.